src/notifications/notificationmanager.cpp

```diff
@@ -210,6 +217,31 @@ void NotificationManager::CloseNotification(uint id, NotificationClosedReason cl
     }
 }
 
+void NotificationManager::MarkNotificationDisplayed(uint id)
+{
+    if (notifications.contains(id)) {
+        const LipstickNotification *notification = notifications.value(id);
```

Comment:
Does the map contain QPointer or raw pointer values?  If the value can be null, the unchecked dereference below could be unsafe